jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -87,6 +88,31 @@ async def job_submission_create(
     return new_job_submission
 
 
+@router.post(
+    "/clone/{id}",
+    status_code=status.HTTP_201_CREATED,
+    response_model=JobSubmissionDetailedView,
+    description="Endpoint for cloning a job script to a new entry owned by the user",
```

Comment:
The description should probably let the caller know that this is used for resubmission and that the new, cloned submission will be submitted to the cluster.